app/models/group.rb

```diff
@@ -56,7 +56,7 @@
 
 class Group < ActiveRecord::Base # rubocop:disable Metrics/ClassLength
   SEARCHABLE_ATTRS = [:name, :short_name, :email, :address, :zip_code, :town, :country,
-    {parent: [:name, :short_name], phone_numbers: [:number],
```

Comment:
Searching groups based on the parent didn't work, with `PostgreSql` or `MySql`, if this is really something we need, we have to use a custom join query in our search method